clients/drcachesim/common/trace_entry.h

```diff
@@ -613,9 +613,9 @@ typedef enum {
     TRACE_MARKER_TYPE_CONTEXT_SWITCH_END,
 
     /**
-     * The marker is used to indicate the current vector length in bytes for
-     * architectures with a dynamic vector length, for example the AArch64 SVE vector
-     * length. This does not apply to fixed length vector architectures.
+     * This marker's value is the current vector length in bytes for architectures with a
+     * dynamic vector length, for example the AArch64 SVE vector length. This does not
```

Comment:
Also mention that the vector length has thread-local scope.

Reply:
Done.